### PR TITLE
Document how to install for Fedora

### PR DESCRIPTION
**Pull request by hegjon**
Install from COPR for now

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,18 @@ The official package uses Lua 5.4.
 Alternatively, a package that uses LuaJIT may be built manually from the [Arch User Repository][aur] using [sile-luajit][aur-sile-luajit].
 A VCS package is also available as [sile-git][aur-sile-git] to build from the latest Git commit.
 
+#### Fedora
+
+A [COPR][copr] repository is available for Fedora users with packages of SILE
+and all the necessary dependencies including fonts.
+Fedora 36 and Fedora 37 are supported. There is work in the progress to get the
+packages added to the official Fedora repository.
```
Comment (alerque) [inline README.md:77]:
```suggestion
Fedora 36 and Fedora 37 are supported.
There is work in progress to get the packages added to the official Fedora repository.
```

File: README.md [hunk 2]
```diff
@@ -69,6 +69,18 @@ The official package uses Lua 5.4.
...
+
+```console
+$ sudo dnf copr enable jonny/SILE
+$ sudo dnf install sile
```
Comment (alerque) [inline README.md:81]: Lets not suggest sudo here, we kind of expect users to be able to adapt to their own system admin style whether that is a root shell (yuck) or sudo or doas or whatever. We don't suggest this for other distros either.

```suggestion
$ dnf copr enable jonny/SILE
$ dnf install sile
```

**Review by alerque — CHANGES_REQUESTED** [review]
Lets add this same info to the manual too. Do you want me to cover that?

**Comment by hegjon** [comment]
> Lets add this same info to the manual too. Do you want me to cover that?

Yes, I did not think about the manual

**Pull request merged**